07-React/react-boilerplate-master/components/input.js

```diff
@@ -0,0 +1,58 @@
+var React = require ('react');
+var _ = require ('lodash');
+
+var Input = React.createClass({
+
+    propTypes:{
+        defaultValue: React.PropTypes.string,
+        onChange: React.PropTypes.func,
+        value: React.PropTypes.oneOfType([
+            React.PropTypes.string,
+            React.PropTypes.number
+        ])
+    },
+
+    getInitialState: function () {
+        var initialState = {};
+
+        if ((_.isUndefined(this.props.value))) {
```

Comment:
**Minor:** no need for double set of parenthesis, you can do ``if (_.isUndefined(this.props.value)) { ... }``.